From c723b649ef34dea4c9f9b3c33c2e4cf533636060 Mon Sep 17 00:00:00 2001
From: Bastian Germann <bastiangermann@fishpost.de>
Date: Sat, 11 Jan 2020 15:27:06 +0100
Subject: [PATCH] doc: build swupdate-sysrestart man page

Signed-off-by: Bastian Germann <bastiangermann@fishpost.de>
---
 doc/source/conf.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 6341500..3141f85 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -238,6 +238,7 @@ man_pages = [
         ('swupdate-client', 'client', u'SWUpdate client tool', '', 1),
         ('swupdate-hawkbitcfg', 'hawkbitcfg', u'SWUpdate Hawkbit Configuration tool', '', 1),
         ('swupdate-progress', 'progress', u'SWUpdate progress tool', '', 1),
+        ('swupdate-sysrestart', 'sysrestart', u'SWUpdate sysrestart tool', '', 1),
         ]
 

